sources/core/Xenko.Core.Mathematics/Half2.cs

```diff
@@ -175,13 +175,33 @@ public override bool Equals(object obj)
         }
 
         /// <summary>
-        /// Performs an explicit conversion from <see cref="Vector3"/> to <see cref="Half3"/>.
+        /// Performs an explicit conversion from <see cref="Half2"/> to <see cref="Vector2"/>.
         /// </summary>
         /// <param name="value">The value.</param>
         /// <returns>The result of the conversion.</returns>
         public static explicit operator Vector2(Half2 value)
         {
             return new Vector2(value.X, value.Y);
         }
+
```

Comment:
Let's move over all those new implicit operators from `HalfX`/`VectorX` files to `DoubleX` file so that `DoubleX` files are self contained.